{{cookiecutter.folder_name}}/Makefile

```diff
@@ -36,12 +37,17 @@ tabdir=./tables
 tables=$(tabdir)/*.tex
 styles=./texmf/styles/*.{sty,cls}
 bibs=./texmf/bib/*.bst
-source=$(main).{tex,bbl,bib} lsstdesc.bib macros.tex authors.tex acknowledgments.tex
+source=$(main).{tex,bbl,bib} lsstdesc.bib authors.tex acknowledgments.tex
 
 tarfiles=$(figures) $(tables) $(styles) $(bibs) $(source)
 
-# Interpret `make` with no target as `make tex`, a latex Note
-all: export flag = \def\flag{tex}
+# Interpret `make` with no target as `make tex`, a latex Note.
+# At present, if the default_format is anything other than
+# [apj|apjl|mnras|prd|prl|emulateapj], a latex Note is made.
+# In future, we could think of using `make` to eg run the ipynb 
+# notebook and make PDF from the output, but this has not been
+# implemented yet.
+all: export flag=\def\flag{$(default)}
```

Comment:
I think this is fine for now, but I expect this to change a bit. We are going to need a `$journal` variable (though `$journal` doesn't make sense for notes, so maybe `$style`) that we pass around to various targets (both `main` and `authlist`).